packages/clay-css/src/scss/variables/_stickers.scss

```diff
@@ -402,6 +402,106 @@ $sticker-dark: map-deep-merge(
 	$sticker-dark
 );
 
+$sticker-outline-primary: () !default;
+$sticker-outline-primary: map-deep-merge(
```

Comment:
I don't think we should be calling this primary, secondary, or warning, etc... These don't directly use the normal colors we would have for those names. These are just additional accent colors so it's probably simpler if we just use numbers for the variants: ie: sticker-outline-0

What do you think?